aicsimageio/writers/ome_zarr_writer.py

```diff
@@ -193,13 +202,19 @@ def write_image(
         xdimindex = dimension_order.find("X")
```

Comment:
Could these use the DimensionNames constant too?

Reply:
resolved in [cdaa92e](https://github.com/AllenCellModeling/aicsimageio/pull/519/commits/cdaa92e40801f76f16fb7ccb4bef2e6f532eb54a)